pkg/prometheus/promcfg.go

```diff
@@ -578,6 +590,75 @@ func (cg *configGenerator) generatePodMonitorConfig(
 	return cfg
 }
 
+func (cg *configGenerator) generateBlackboxMonitorConfig(version semver.Version, m *v1.BlackboxMonitor) yaml.MapSlice {
```

Comment:
I think this now needs to also accept the following parameters to be compliant with PodMonitor and ServiceMonitor scrape generators:
```
- overrideHonorLabels
- overrideHonorTimestamps
- ignoreNamespaceSelectors
- enforcedNamespaceLabel
```

Reply:
made it compliant